R/gen_tibble_vcf.R

```diff
@@ -6,10 +6,17 @@ gen_tibble_vcf <- function(x, ..., parser = "cpp",
                            backingfile = NULL, quiet = FALSE) {
   parser <- match.arg(parser, c("vcfR", "cpp"))
   if (parser == "cpp"){
-    rds_path <- vcf_to_fbm_cpp(x,
-                                backingfile = backingfile,
-                                chunk_size = chunk_size,
-                                quiet = quiet)
+    if(.Platform$OS.type!="windows"){
```

Comment:
I would be better to change the parser depending on OS, so, that, if os=="windows", parser="vcfR". That would minimise copying and pasting of code.